fix(communities)_: token master can have missed revealed members addresses

mprakhov opened the pull request:
In the scope of this PR done:
1. Moved tests related to sharing accounts to a separate test suite. Added more tests related to sharing accounts to the token master
2. If TokenMaster restores his account by a seed phrase - send him requests to join with revealed addresses
3. If a member edits the shared address, the token master receives it only when a control node approves the new addresses
4. Fixed bug in integration tests, where `GetBalancesByChain` returned all known balances, but not for certain accounts and chain ids
5. Filter our outdated edit shared address and privileged user sync messages
6. Fixed changing privileged permissions (TM -> admin, admin -> TM)
7. Fixed proper usage of `LastUpdateClock` for the `CommunityMember`
8. Fixed `createOnRequestCommunity` test util, which always created a community with `AUTO_REQUEST` 

Closes #https://github.com/status-im/status-desktop/issues/15139

IMPACT: 
- requests to join the community
- sharing revealed accounts to the token master

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (107)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | 756f77a8 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/1/) | 2024-06-19 16:03:16 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 756f77a85fdd43ce9f2c018eaf51830c5f3dba56 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/1/) | 2024-06-19 16:04:28 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240619-160023-756f77a85fdd43ce9f2c018eaf51830c5f3dba56-pr5383.zip) |
| :heavy_check_mark: | 756f77a85fdd43ce9f2c018eaf51830c5f3dba56 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/1/) | 2024-06-19 16:05:29 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240619-160023-756f77a85fdd43ce9f2c018eaf51830c5f3dba56-pr5383.zip) |
| :heavy_check_mark: | 756f77a85fdd43ce9f2c018eaf51830c5f3dba56 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/1/) | 2024-06-19 16:06:11 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240619-160023-756f77a85fdd43ce9f2c018eaf51830c5f3dba56-pr5383.aar) |
| | | | | | | |
| :heavy_multiplication_x: | 3d0c7905 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/2/) | 2024-06-19 16:05:17 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 4438772e413d3aa216bc86c3f0845e1b47f23bb0 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/2/) | 2024-06-19 16:06:59 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240619-160445-4438772e413d3aa216bc86c3f0845e1b47f23bb0-pr5383.zip) |
| :heavy_check_mark: | 4438772e413d3aa216bc86c3f0845e1b47f23bb0 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/2/) | 2024-06-19 16:08:42 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240619-160632-4438772e413d3aa216bc86c3f0845e1b47f23bb0-pr5383.aar) |
| :heavy_check_mark: | 4438772e413d3aa216bc86c3f0845e1b47f23bb0 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/2/) | 2024-06-19 16:11:03 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240619-160546-4438772e413d3aa216bc86c3f0845e1b47f23bb0-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 4438772e | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/3/) | 2024-06-19 16:07:00 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d0f1c83c09997a5735c41e27b4c76d8944c1232d | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/3/) | 2024-06-19 16:23:41 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240619-162158-d0f1c83c09997a5735c41e27b4c76d8944c1232d-pr5383.aar) |
| :heavy_check_mark: | d0f1c83c09997a5735c41e27b4c76d8944c1232d | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/3/) | 2024-06-19 16:24:54 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240619-162202-d0f1c83c09997a5735c41e27b4c76d8944c1232d-pr5383.zip) |
| :heavy_check_mark: | d0f1c83c09997a5735c41e27b4c76d8944c1232d | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/3/) | 2024-06-19 16:26:02 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240619-162202-d0f1c83c09997a5735c41e27b4c76d8944c1232d-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | d0f1c83c | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/4/) | 2024-06-19 17:05:11 | ~43 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/4//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | 22854278 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/5/) | 2024-06-20 11:17:33 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 22854278400d90992626c3ebc02ca86c72415a40 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/4/) | 2024-06-20 11:18:10 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240620-111608-22854278400d90992626c3ebc02ca86c72415a40-pr5383.aar) |
| :heavy_check_mark: | 22854278400d90992626c3ebc02ca86c72415a40 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/4/) | 2024-06-20 11:18:20 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240620-111608-22854278400d90992626c3ebc02ca86c72415a40-pr5383.zip) |
| :heavy_check_mark: | 22854278400d90992626c3ebc02ca86c72415a40 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/4/) | 2024-06-20 11:20:18 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240620-111608-22854278400d90992626c3ebc02ca86c72415a40-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | d59c147b | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/6/) | 2024-06-20 11:20:27 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d59c147ba0feaa593bc9d7a5675532babc85d5b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/5/) | 2024-06-20 11:21:06 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240620-111848-d59c147ba0feaa593bc9d7a5675532babc85d5b4-pr5383.aar) |
| :heavy_check_mark: | d59c147ba0feaa593bc9d7a5675532babc85d5b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/5/) | 2024-06-20 11:21:18 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240620-111852-d59c147ba0feaa593bc9d7a5675532babc85d5b4-pr5383.zip) |
| :heavy_check_mark: | d59c147ba0feaa593bc9d7a5675532babc85d5b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/5/) | 2024-06-20 11:23:22 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240620-112034-d59c147ba0feaa593bc9d7a5675532babc85d5b4-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 1f3e12b1 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/7/) | 2024-06-20 15:39:38 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 1f3e12b1738f833248557e60da9c6b6bd0ca1e4f | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/6/) | 2024-06-20 15:40:11 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240620-153803-1f3e12b1738f833248557e60da9c6b6bd0ca1e4f-pr5383.aar) |
| :heavy_check_mark: | 1f3e12b1738f833248557e60da9c6b6bd0ca1e4f | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/6/) | 2024-06-20 15:40:24 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240620-153803-1f3e12b1738f833248557e60da9c6b6bd0ca1e4f-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 2fc53e36 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/8/) | 2024-06-20 15:44:18 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/8//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 2fc53e36783a03548e694a3cd759c7c48b1577ea | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/7/) | 2024-06-20 15:44:32 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240620-154304-2fc53e36783a03548e694a3cd759c7c48b1577ea-pr5383.aar) |
| :heavy_check_mark: | 2fc53e36783a03548e694a3cd759c7c48b1577ea | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/7/) | 2024-06-20 15:45:03 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240620-154304-2fc53e36783a03548e694a3cd759c7c48b1577ea-pr5383.zip) |
| :heavy_check_mark: | 2fc53e36783a03548e694a3cd759c7c48b1577ea | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/7/) | 2024-06-20 15:51:26 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240620-154833-2fc53e36783a03548e694a3cd759c7c48b1577ea-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 8dfcbdd15dd4f0f49265ea0b2979e30e99aaee4e | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/8/) | 2024-06-20 15:52:18 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240620-155004-8dfcbdd15dd4f0f49265ea0b2979e30e99aaee4e-pr5383.zip) |
| :heavy_check_mark: | 8dfcbdd15dd4f0f49265ea0b2979e30e99aaee4e | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/8/) | 2024-06-20 15:56:04 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240620-155004-8dfcbdd15dd4f0f49265ea0b2979e30e99aaee4e-pr5383.aar) |
| | | | | | | |
| :heavy_check_mark: | 8dfcbdd1 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/9/) | 2024-06-20 16:33:25 | ~43 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8daefaf80efaebda6f5257c218a5c5822a4bfc72 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/9/) | 2024-06-20 17:40:27 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240620-173811-8daefaf80efaebda6f5257c218a5c5822a4bfc72-pr5383.aar) |
| :heavy_check_mark: | 8daefaf80efaebda6f5257c218a5c5822a4bfc72 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/9/) | 2024-06-20 17:40:29 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240620-173815-8daefaf80efaebda6f5257c218a5c5822a4bfc72-pr5383.zip) |
| :heavy_check_mark: | 8daefaf80efaebda6f5257c218a5c5822a4bfc72 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/10/) | 2024-06-20 17:41:21 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240620-173815-8daefaf80efaebda6f5257c218a5c5822a4bfc72-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 8daefaf8 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/10/) | 2024-06-20 18:22:33 | ~44 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 2a95029b40bcb08024cdefab84cbdb80512f5a0e | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/10/) | 2024-06-21 08:28:41 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240621-082628-2a95029b40bcb08024cdefab84cbdb80512f5a0e-pr5383.aar) |
| :heavy_check_mark: | 2a95029b40bcb08024cdefab84cbdb80512f5a0e | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/10/) | 2024-06-21 08:28:52 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240621-082632-2a95029b40bcb08024cdefab84cbdb80512f5a0e-pr5383.zip) |
| :heavy_check_mark: | 2a95029b40bcb08024cdefab84cbdb80512f5a0e | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/11/) | 2024-06-21 08:29:15 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240621-082628-2a95029b40bcb08024cdefab84cbdb80512f5a0e-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 6faceff8eea9ac3912c73f399ecd5b40e2672b7f | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/12/) | 2024-06-21 09:05:42 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240621-090133-6faceff8eea9ac3912c73f399ecd5b40e2672b7f-pr5383.zip) |
| :heavy_check_mark: | 6faceff8eea9ac3912c73f399ecd5b40e2672b7f | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/11/) | 2024-06-21 09:07:08 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240621-090133-6faceff8eea9ac3912c73f399ecd5b40e2672b7f-pr5383.aar) |
| :heavy_check_mark: | 6faceff8eea9ac3912c73f399ecd5b40e2672b7f | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/11/) | 2024-06-21 09:07:18 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240621-090133-6faceff8eea9ac3912c73f399ecd5b40e2672b7f-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 2a95029b | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/11/) | 2024-06-21 09:11:58 | ~45 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/11//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6faceff8 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/12/) | 2024-06-21 09:56:52 | ~44 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 9444309679a5a54afb734fcf625c7faff36157e3 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/12/) | 2024-06-24 17:31:45 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240624-172918-9444309679a5a54afb734fcf625c7faff36157e3-pr5383.zip) |
| :heavy_check_mark: | 9444309679a5a54afb734fcf625c7faff36157e3 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/12/) | 2024-06-24 17:31:46 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240624-172914-9444309679a5a54afb734fcf625c7faff36157e3-pr5383.aar) |
| :heavy_check_mark: | 9444309679a5a54afb734fcf625c7faff36157e3 | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/13/) | 2024-06-24 17:32:14 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240624-172918-9444309679a5a54afb734fcf625c7faff36157e3-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 94443096 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/13/) | 2024-06-24 18:06:23 | ~37 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/13//consoleText) |
| :heavy_multiplication_x: | 94443096 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/14/) | 2024-06-25 08:49:56 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/14//consoleText) |
| :heavy_multiplication_x: | 94443096 | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/15/) | 2024-06-25 10:32:33 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/15//consoleText) |
| :heavy_multiplication_x: | 94443096 | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/16/) | 2024-06-25 18:15:28 | ~37 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/16//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 287aed9c0893ba6266b15df4ff73fa52e2766b0c | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/14/) | 2024-06-26 10:49:12 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240626-104535-287aed9c0893ba6266b15df4ff73fa52e2766b0c-pr5383.zip) |
| :heavy_check_mark: | 287aed9c0893ba6266b15df4ff73fa52e2766b0c | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/13/) | 2024-06-26 10:50:20 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240626-104534-287aed9c0893ba6266b15df4ff73fa52e2766b0c-pr5383.aar) |
| :heavy_check_mark: | 287aed9c0893ba6266b15df4ff73fa52e2766b0c | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/13/) | 2024-06-26 10:50:55 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240626-104535-287aed9c0893ba6266b15df4ff73fa52e2766b0c-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | 287aed9c | [#17](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/17/) | 2024-06-26 11:26:11 | ~40 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/17//consoleText) |
| :heavy_multiplication_x: | 287aed9c | [#18](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/18/) | 2024-06-26 12:17:09 | ~38 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/18//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a8fc9c99ce25706b0162c108478b4a7099f86ff3 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/14/) | 2024-06-26 12:59:40 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240626-125721-a8fc9c99ce25706b0162c108478b4a7099f86ff3-pr5383.zip) |
| :heavy_check_mark: | a8fc9c99ce25706b0162c108478b4a7099f86ff3 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/15/) | 2024-06-26 13:00:33 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240626-125721-a8fc9c99ce25706b0162c108478b4a7099f86ff3-pr5383.zip) |
| :heavy_check_mark: | a8fc9c99ce25706b0162c108478b4a7099f86ff3 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/14/) | 2024-06-26 13:01:59 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240626-125721-a8fc9c99ce25706b0162c108478b4a7099f86ff3-pr5383.aar) |
| | | | | | | |
| :heavy_check_mark: | a8fc9c99 | [#19](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/19/) | 2024-06-26 13:39:39 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/19//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 55ee1b7aeeccb95f870e0ff864e9320ad8af273f | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/15/) | 2024-06-26 15:17:56 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240626-151536-55ee1b7aeeccb95f870e0ff864e9320ad8af273f-pr5383.aar) |
| :heavy_check_mark: | 55ee1b7aeeccb95f870e0ff864e9320ad8af273f | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/16/) | 2024-06-26 15:18:27 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240626-151536-55ee1b7aeeccb95f870e0ff864e9320ad8af273f-pr5383.zip) |
| :heavy_check_mark: | 55ee1b7aeeccb95f870e0ff864e9320ad8af273f | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/15/) | 2024-06-26 15:18:43 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240626-151536-55ee1b7aeeccb95f870e0ff864e9320ad8af273f-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 55ee1b7a | [#20](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/20/) | 2024-06-26 16:01:21 | ~45 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/20//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 72f92b2de3915f3ad3df2ff162aa3e00ed509565 | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/16/) | 2024-06-27 07:50:01 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240627-074743-72f92b2de3915f3ad3df2ff162aa3e00ed509565-pr5383.aar) |
| :heavy_check_mark: | 72f92b2de3915f3ad3df2ff162aa3e00ed509565 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/16/) | 2024-06-27 07:51:34 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240627-074743-72f92b2de3915f3ad3df2ff162aa3e00ed509565-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 72f92b2d | [#21](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/21/) | 2024-06-27 08:30:08 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/21//consoleText) |
| | | | | | | |
| :heavy_check_mark: | d7dec1ce | [#17](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/17/) | 2024-06-27 09:25:04 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240627-092239-d7dec1-pr5383.aar) |
| :heavy_check_mark: | d7dec1ce | [#17](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/17/) | 2024-06-27 09:25:09 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240627-092239-d7dec1-pr5383.zip) |
| :heavy_check_mark: | d7dec1ce | [#19](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/19/) | 2024-06-27 09:26:45 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240627-092239-d7dec1-pr5383.zip) |
| :heavy_check_mark: | d7dec1ce | [#22](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/22/) | 2024-06-27 10:05:17 | ~42 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/22//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 915c67b6 | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/18/) | 2024-07-01 20:38:19 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240701-203557-915c67-pr5383.zip) |
| :heavy_check_mark: | 915c67b6 | [#20](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/20/) | 2024-07-01 20:39:11 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240701-203557-915c67-pr5383.zip) |
| :heavy_check_mark: | 915c67b6 | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/18/) | 2024-07-01 20:40:10 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240701-203553-915c67-pr5383.aar) |
| :heavy_check_mark: | 915c67b6 | [#23](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/23/) | 2024-07-01 21:19:33 | ~43 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/23//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a74a6051 | [#19](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/19/) | 2024-07-03 13:53:55 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240703-135127-a74a60-pr5383.zip) |
| :heavy_check_mark: | a74a6051 | [#21](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/21/) | 2024-07-03 13:54:22 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240703-135127-a74a60-pr5383.zip) |
| :heavy_check_mark: | a74a6051 | [#19](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/19/) | 2024-07-03 13:55:59 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240703-135127-a74a60-pr5383.aar) |
| :heavy_check_mark: | a74a6051 | [#24](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/24/) | 2024-07-03 14:35:12 | ~43 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/24//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 16ff7f19 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/1/) | 2024-07-05 12:18:38 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/1//consoleText) |
| :heavy_check_mark: | 16ff7f19 | [#22](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/22/) | 2024-07-05 12:19:04 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-121554-16ff7f-pr5383.zip) |
| :heavy_check_mark: | 16ff7f19 | [#20](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/20/) | 2024-07-05 12:19:34 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-121554-16ff7f-pr5383.zip) |
| :heavy_check_mark: | 16ff7f19 | [#20](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/20/) | 2024-07-05 12:21:04 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-121551-16ff7f-pr5383.aar) |
| :heavy_multiplication_x: | 16ff7f19 | [#25](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/25/) | 2024-07-05 12:23:18 | ~7 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/25//consoleText) |
| :heavy_multiplication_x: | 16ff7f19 | [#26](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/26/) | 2024-07-05 12:40:44 | ~8 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/26//consoleText) |
| | | | | | | |
| :heavy_multiplication_x: | a2f12301 | [#27](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/27/) | 2024-07-05 13:45:39 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/27//consoleText) |
| :heavy_check_mark: | a2f12301 | [#21](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/21/) | 2024-07-05 13:46:43 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-134427-a2f123-pr5383.aar) |
| :heavy_check_mark: | a2f12301 | [#21](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/21/) | 2024-07-05 13:46:51 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-134431-a2f123-pr5383.zip) |
| :heavy_check_mark: | a2f12301 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/2/) | 2024-07-05 13:47:15 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/2//consoleText) |
| :heavy_check_mark: | a2f12301 | [#23](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/23/) | 2024-07-05 13:47:34 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-134431-a2f123-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | c76677ce | [#28](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/28/) | 2024-07-05 13:46:51 | ~56 sec | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/28//consoleText) |
| :heavy_check_mark: | c76677ce | [#22](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/22/) | 2024-07-05 13:48:57 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-134700-c76677-pr5383.aar) |
| :heavy_check_mark: | c76677ce | [#22](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/22/) | 2024-07-05 13:49:17 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-134710-c76677-pr5383.zip) |
| :heavy_check_mark: | c76677ce | [#24](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/24/) | 2024-07-05 13:50:36 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-134750-c76677-pr5383.zip) |
| | | | | | | |
| :heavy_multiplication_x: | d83ac7b8 | [#29](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/29/) | 2024-07-05 13:51:56 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/29//consoleText) |
| :heavy_check_mark: | d83ac7b8 | [#23](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/23/) | 2024-07-05 13:53:01 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-135033-d83ac7-pr5383.aar) |
| :heavy_check_mark: | d83ac7b8 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/4/) | 2024-07-05 13:53:16 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/4//consoleText) |
| :heavy_check_mark: | d83ac7b8 | [#23](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/23/) | 2024-07-05 13:54:03 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-135033-d83ac7-pr5383.zip) |
| :heavy_check_mark: | d83ac7b8 | [#25](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/25/) | 2024-07-05 13:54:41 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-135056-d83ac7-pr5383.zip) |
| | | | | | | |
| :heavy_check_mark: | 5f04be6c | [#24](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/24/) | 2024-07-05 14:03:35 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-140209-5f04be-pr5383.aar) |
| :heavy_check_mark: | 5f04be6c | [#24](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/24/) | 2024-07-05 14:04:11 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-140209-5f04be-pr5383.zip) |
| :heavy_check_mark: | 5f04be6c | [#26](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/26/) | 2024-07-05 14:05:01 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-140209-5f04be-pr5383.zip) |
| :heavy_multiplication_x: | 5f04be6c | [#30](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/30/) | 2024-07-05 14:09:02 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/30//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 451ac25d | [#25](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/25/) | 2024-07-05 14:08:01 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240705-140629-451ac2-pr5383.aar) |
| :heavy_check_mark: | 451ac25d | [#25](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/25/) | 2024-07-05 14:08:27 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240705-140631-451ac2-pr5383.zip) |
| :heavy_check_mark: | 451ac25d | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/7/) | 2024-07-05 14:09:07 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/7//consoleText) |
| :heavy_check_mark: | 451ac25d | [#27](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/27/) | 2024-07-05 14:09:24 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240705-140631-451ac2-pr5383.zip) |
| :heavy_multiplication_x: | 451ac25d | [#31](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/31/) | 2024-07-05 14:15:45 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/31//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 6411279b | [#26](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/26/) | 2024-07-09 10:53:54 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240709-105147-641127-pr5383.aar) |
| :heavy_check_mark: | 6411279b | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/8/) | 2024-07-09 10:54:22 | ~2 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-5383/8//consoleText) |
| :heavy_check_mark: | 6411279b | [#28](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/28/) | 2024-07-09 10:54:48 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240709-105151-641127-pr5383.zip) |
| :heavy_check_mark: | 6411279b | [#26](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/26/) | 2024-07-09 10:57:09 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240709-105151-641127-pr5383.zip) |
| :heavy_multiplication_x: | 6411279b | [#32](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/32/) | 2024-07-09 10:59:20 | ~7 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/32//consoleText) |
| :heavy_check_mark: | 6411279b | [#33](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/33/) | 2024-07-09 12:05:10 | ~44 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/33//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 91590576 | [#27](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-5383/27/) | 2024-07-09 15:25:39 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240709-152348-915905-pr5383.aar) |
| :heavy_check_mark: | 91590576 | [#29](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-5383/29/) | 2024-07-09 15:26:44 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240709-152348-915905-pr5383.zip) |
| :heavy_check_mark: | 91590576 | [#27](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-5383/27/) | 2024-07-09 15:28:44 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240709-152353-915905-pr5383.zip) |
| :heavy_check_mark: | 91590576 | [#34](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/34/) | 2024-07-09 16:07:13 | ~43 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-5383/34//consoleText) |

jrainville commented on protocol/communities/communnity_privileged_member_sync_msg.go:
```diff
@@ -37,18 +46,14 @@ func (m *Manager) HandleRequestToJoinPrivilegedUserSyncMessage(message *protobuf
-			}
+		err := m.processPrivilegedUserSharedRequestToJoin(community, requestToJoin)
+		if err != nil {
+			m.logger.Info("error to handle shared request to join",
```
Should this be `Warn`?

jrainville commented on protocol/communities/communnity_privileged_member_sync_msg.go:
```diff
@@ -57,29 +62,26 @@ func (m *Manager) HandleRequestToJoinPrivilegedUserSyncMessage(message *protobuf
-			}
+		err := m.processPrivilegedUserSharedRequestToJoin(community, requestToJoin)
+		if err != nil {
+			m.logger.Info("error to handle shared request to join from sync all requests to join msg",
```
Same here

jrainville commented on protocol/communities/communnity_privileged_member_sync_msg.go:
```diff
@@ -88,3 +90,56 @@ func (m *Manager) HandleSyncAllRequestToJoinForNewPrivilegedMember(message *prot
+	}
+
+	member, exists := community.Members()[requestToJoin.PublicKey]
+	if exists && existingRequestToJoin != nil && (existingRequestToJoin.Clock > requestToJoin.Clock || member.LastUpdateClock > requestToJoin.Clock) {
```
Wouldn't `existingRequestToJoin` be `nil` if we have `ErrNoRows` above? If so, we probably have to guard against it here to avoid a crash

mprakhov replied:
but we do it already:
`if exists && _**existingRequestToJoin != nil**_ && (existingRequestToJoin.Clock > requestToJoin.Clock || member.LastUpdateClock > requestToJoin.Clock)`

further, `existingRequestToJoin` is not used

jrainville commented on protocol/communities/errors.go:
```diff
@@ -50,3 +50,5 @@ var ErrNoFreeSpaceForHistoryArchives = errors.New("history archive: No free spac
 var ErrPermissionToJoinNotSatisfied = errors.New("permission to join not satisfied")
 var ErrBannedMemberNotFound = errors.New("banned member not found")
 var ErrGrantMemberPublicKeyIsDifferent = errors.New("grant member public key is different")
+var ErrEditSharedAddressesRequestOutdated = errors.New("outdated edit share addresses request")
```
```suggestion
var ErrEditSharedAddressesRequestOutdated = errors.New("outdated edit shares addresses request")
```

jrainville commented on protocol/communities/manager.go:
```diff
@@ -970,6 +970,11 @@ func (rmr reevaluateMemberRole) hasChangedToPrivileged() bool {
 	return rmr.hasChanged() && rmr.old == protobuf.CommunityMember_ROLE_NONE
 }
 
+func (rmr reevaluateMemberRole) hasChangedPrivileged() bool {
```
```suggestion
func (rmr reevaluateMemberRole) hasChangedPrivilegedRole() bool {
```

jrainville reviewed (commented):
Nice job.

Some small questions, but also I think I found an error

friofry commented:
I've gone through the code twice. It looks good to me. But I don't know the whole mechanism of how the collection of the revealed address works. I'll try again tomorrow.

mprakhov committed:
fix(CommunityMember)_: update LastUpdateClock

mprakhov committed:
fix(reevaluateMembers)_: changing privileged permissions (TM -> admin, admin -> TM)

mprakhov committed:
fix(TokenBalanceMock)_: mocked GetBalancesByChain returns all known balances, but not for certain accounts and chain ids

mprakhov committed:
fix(TestUtils)_: remove hardcoded CommunityPermissions_AUTO_ACCEPT for createCommunityConfigurable Membership

mprakhov committed:
fix(CommunitySharedAddresses)_:
- TokenMaster request shared addresses on restoring from a backup
- Fixes sharing members revealed adresses
- Filter out outdated ApplicationMetadataMessage_COMMUNITY_PRIVILEGED_USER_SYNC_MESSAGE messages

mprakhov commented:
@jrainville , @friofry just in case, I extracted bugfixes of found bugs during bugfix in this ticket into the separate commits :)

jrainville reviewed (approved):
Nice job

friofry commented on protocol/communities/communnity_privileged_member_sync_msg.go:
```diff
@@ -88,3 +94,80 @@ func (m *Manager) HandleSyncAllRequestToJoinForNewPrivilegedMember(message *prot
+		}
+	}
+
+	if member, memberExists := community.Members()[requestToJoin.PublicKey]; memberExists && member.LastUpdateClock > requestToJoin.Clock {
```
maybe move this above the first `if statusupdate`?

mprakhov replied:
ty, fixed

friofry reviewed (approved):
Looks good to me! I leared a lot about communities :)
